LunaDll/LuaMain/LuaProxyComponent/LuaProxyAudio.cpp

```diff
@@ -175,24 +175,24 @@ double LuaProxy::Audio::MusicGetLoopLength()
 
 
 
-void LuaProxy::Audio::MusicRewind()
+void LuaProxy::Audio::MusicRestart()
```

Comment:
Little note why the function is called "Rewind": it just puts carret to the begining of the file with no matter does the music plays or not. It's not an actual "restart", it only puts carret into begining of the file. So, if music plays, it works as "restart", if music does NOT playes (the file is open, but it doesn't play), it won't start it's playback, it just puts carret to begining without other actions.

Reply:
Fair point about that. I'd agree with it being `MusicRewind` across the board.